src/App.js

```diff
@@ -1,12 +1,23 @@
 import React, {Component} from 'react';
 import NavBar from "./components/Nav";
+import {BrowserRouter as Router, Route} from 'react-router-dom';
+
+const Home = () => (
+    <div>
+        <NavBar/>
+    </div>
+)
 
 class App extends Component {
     render() {
         return (
-            <div>
-                <NavBar/>
-            </div>
+            <Router>
+                <div>
+                    <div>
```

Comment:
Not sure why all of these nested `<div>`s are here. You will also want `<NavBar />` and any other common components that appear on every page to be here, not inside your routes. E.g.:

```jsx
<Router>
    <NavBar />
    <Container>
        <Route exact path="/" component={Home} />
        {/* Other Routes Here */}
    </Container>
</Router>
```